tasks.js

```diff
@@ -12,41 +13,72 @@ module.exports = (resources) => {
     let output = {}
 
     try {
-      const data = await setup(await validate( await load(input)))
-      // business logic goes here...
-      // output = ...
-    } catch (e) { 
-      // if (e.message === ...
-      //   throw new Error('...
-      // }
-      // Default case
-      _debug(e.stack)
-      throw (e)
+      const data = await setup(await validate(await load(input)))
+      output = data.files + data.errored_files
+      //upload file to S3
+        // s3_driver.add(shared.PATH, output).then(response => {
+        //   console.log(shared.OK, response);
+        // })
+        //   .catch(error => {
+        //     console.error(shared.ERROR, error);
+        // });
```

Comment:
If the s3_driver fails then we would want the task to return an error, but in this case it would fail silently and the task will return as if it was successful